src/analysis/retail/priest/holy/modules/features/StatValuesSpellInfo.tsx

```diff
@@ -112,7 +112,7 @@ export default {
     mastery: false,
     vers: true,
   },
-  [SPELLS.TRAIL_OF_LIGHT_HEAL.id]: {
+  [SPELLS.TRAIL_OF_LIGHT_TALENT_HEAL.id]: {
```

Comment:
i assume that these are separate heal IDs from the actual talent spell?

Reply:
Yes